peachjam/templates/account/password_reset_from_key_done.html

```diff
@@ -1,4 +1,5 @@
 {% extends "account/base.html" %}
+{% load i18n %}
```

Comment:
the i18n tags are for templates that have translation strings, this one doesn't atm. But there are some strings that need to be tagged for translation like the `Reset password` etc

Reply:
Missed that. I'll update.